src/main/java/hudson/plugins/sonar/SonarGlobalConfiguration.java

```diff
@@ -52,9 +53,9 @@
  * The global configuration was migrated from SonarPublisher to this component.
  */
 @Extension(ordinal = 100)
-public class SonarGlobalConfiguration extends GlobalConfiguration {
+public class SonarGlobalConfiguration extends GlobalConfiguration implements Serializable {
 
-  private final Supplier<Jenkins> jenkinsSupplier;
+  private final transient Supplier<Jenkins> supplyJenkins;
```

Comment:
Why rename this? I liked the old name much better!

Reply:
Me too, but people already persisted config for that field. So if I do not rename the field, their Jenkins instances will keep trying to deserialise the incorrect into the same field.

Now that it is renamed Jenkins offers them the option to delete the old value, because there is no field with the same name anymore.

Reply:
Fair enough!